packages/core/src/base/typography/_typography.docs.scss

```diff
@@ -18,7 +18,7 @@ Link
 
 Links connect users to a different page or further information.
 
-The only base HTML element the design system applies styling to is the `<a>` element. To prevent this, you can override the `$ds-include-base-html-rulesets` Sass variable and set it to `false`.
+The only base HTML element the design system applies styling to is the `<a>` element. To prevent this, you can override the `$ds-include-base-html-rulesets` Sass variable and set it to `false`. You can also use `.ds-c-link` and `.ds-c-link--inverse` for stlyling links.
```

Comment:
typo here

Reply:
Also, what do you think about tweaking the sentence to be:
 "you can also use ... classes for applying the link styling" 

Slightly different because there may be scenarios where you want to apply link styling to non `<a>` elements

Reply:
Ah I see, I'm trying to think of a scenario where you'd use the link styling which is only text color instead of using a utility class for changing the text color. I would think we would want to encourage that route as opposed to using the link class to style items other than links.

Reply:
Oh right, I think the link styling should also include `text-decoration: underline` as well. There still the issue of when we would want to style a non link element as a link, but personally I think providing that as a small option is useful, especially for writing CSS.